pkg/parser/keywords.go

```diff
@@ -622,6 +622,7 @@ var Keywords = []KeywordsType{
 	{"VALIDATION", false, "unreserved"},
 	{"VALUE", false, "unreserved"},
 	{"VARIABLES", false, "unreserved"},
+	{"VECTOR", false, "unreserved"},
```

Comment:
Why add this?

Reply:
I didn't. It was auto-added on code generation :(

I can remove it, but it will re-add.

Reply:
This has been fixed by a rebase. So someone else's PR added it.